unit/goto-instrument/cover/cover_only.cpp

```diff
@@ -0,0 +1,157 @@
+/*******************************************************************\
+
+Module: Tests for coverage instrumentation config object and filters
+
+Author: Diffblue Ltd, 2019
+
+\*******************************************************************/
+
+#include <goto-instrument/cover.h>
+#include <testing-utils/message.h>
+#include <testing-utils/use_catch.h>
+#include <util/options.h>
+#include <util/symbol_table.h>
+#include <util/ui_message.h>
+
+symbolt create_new_symbol(const irep_idt &name, const irep_idt &file_name)
```

Comment:
:pick: This is the sort of name we could easily use in another place over time, I'd wrap this up in an anonymous namespace to prevent linking errors in the future